pkgs/os-specific/linux/nftables/python.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, setuptools
+, nftables
+}:
+
+buildPythonPackage {
+  pname = "nftables";
+  inherit (nftables) version src meta;
+
+  patches = [ ./fix-py-libnftables.patch ];
```

Comment:
Why bother with a patch when you can just `--replace libnftables.so.1` directly in the postPatch?

Reply:
`--replace` is silent when it fails, a patch will fail to apply if this ever change.

Reply:
And if replace it with `--replace-fail`?

Reply:
TIL, updated, thanks.

Reply:
Needs to delete the file with the patch.